docs/user-guide/cdf_format_guide.rst

```diff
@@ -134,8 +134,8 @@ Section 4 of this document.
      - This is an optional field that may not be needed for all products. Where it is used, identifier should be short (e.q. 3-8 characters) descriptors that are helpful to end-users. If a descriptor contains multiple components, underscores are used to separate those components.
      - An optional time span may be specified as "2s" to represent a data file that spans two seconds. In this case, "10s" and "5m" are other expected values that correspond with ten seconds and 5 minutes respectively.
    * - startTime
-     - The start time of the contained data given in "YYYYMMDD_hhmmss"
-     - `20220601_101520`
+     - The start time of the contained data given in `ISO 8601 <https://en.wikipedia.org/wiki/ISO_8601>`_ format.
+     - `2023-05-19T00:00:03.000`
```

Comment:
According to the guide you referenced and looking at other SPDF files, this is not the format that is used. It is "YYYYMMDDTHHMMSS".

Reply:
Updated to match the format here.